### ISSUE TITLE
Added pulling tasks from localstorage

### ISSUE THREAD

**Issue by mic051** [cse110-sp24-group10/cse110-sp24-group10#88]
Tags not fully implemented as we are waiting on the task list team to finish custom tags. Code is very similar to the one used in the calendar.

**Comment by mic051** [comment]
code is in journal-calendar-link

**Comment by mic051** [comment]
done, code in pranav-tasks and should be shortly merged into main
